src/pudl/output/eia860.py

```diff
@@ -28,6 +28,11 @@ def utilities_eia860(pudl_engine, start_date=None, end_date=None):
         pandas.DataFrame: A DataFrame containing all the fields of the EIA 860
         Utilities table.
     """
+    logger.warning(
+        "pudl.output.eia860.utilities_eia860() will be deprecated in a future version of PUDL."
+        " In the future, call the PudlTabl.utils_eia860() method or pull the denorm_utilities_eia table"
+        "directly from the pudl.sqlite database."
+    )
```

Comment:
We don't need to fix this to merge, but I think we should probably avoid directing folks to use the `PudlTabl` outputs since we intend to deprecate that too -- if we're giving them directions for the future, it should probably be to use the DB tables directly.